virttest/utils_misc.py

```diff
@@ -25,6 +25,12 @@
 import platform
 import traceback
 import math
+import select
+
+try:
+    from io import BytesIO
+except:
```

Comment:
except ImportError:

Reply:
I just saw your comment, yes, I modified same with you comments.
Thanks.